src/cmake.ts

```diff
@@ -464,17 +456,24 @@ class ErrorParser extends util.OutputParser {
     private _activeParser: Maybe<DiagnosticParser>;
     private _parserCollection: Set<DiagnosticParser>;
 
-    constructor(binaryDir: string) {
+    constructor(binaryDir: string, parsers: Maybe<string[]>) {

```

Comment:
Perhapse a variadic list of parsers? Such as:

~~~typescript
constructor(binaryDir: string, ...parsers: string[])
~~~

May be simpler, and doesn't require users to pass in an explicit `null`.